tests/output/InteractionsTest.py

```diff
@@ -12,11 +13,35 @@ class InteractionsTest(unittest.TestCase):
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
         section = Section('')
-        self.assertRaises(TypeError, fail_acquire_settings, log_printer, None,
-                          section)
+        self.assertRaises(TypeError, fail_acquire_settings, log_printer,
+                          None, section)
+
         self.assertRaises(AssertionError,
                           fail_acquire_settings,
                           log_printer,
                           {'setting': ['description', 'bear']}, section)
-        self.assertEqual(fail_acquire_settings(log_printer, {}, section), None,
-                         section)
+
+        self.assertEqual(fail_acquire_settings(log_printer, {}, None),
+                         None, section)
+
+    def test_section_deprecation(self):
+
+        with LogCapture() as capture:
+
+            self.test_()
+            capture.check(
+              ('root',
+               'WARNING',
+               'fail_acquire_settings: section parameter'
+               ' is deprecated.'),
+              ('root',
+               'WARNING',
+               'fail_acquire_settings: section parameter'
+               ' is deprecated.'),
+              ('root',
+               'ERROR',
+               'During execution, we found that some required settings were'
+               ' not provided. '
+               'They are:\n'
+               'setting (from bear) - description')
+                          )
```

Comment:
brace should close on same indent level as opening statement (so aligned to beginning of `capture...`)